tests/ard_reprocessed_l1s/dev_go_reprocess.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+# If a number is passed in it is assumed to be the scene limit
+# otherwise the default is 400
+
+set -o errexit
+set -o xtrace
+
+if [[ "$HOSTNAME" == *"gadi"* ]]; then
+	echo If module load breaks check out a clean environment
+	module use /g/data/v10/public/modules/modulefiles
+	module use /g/data/v10/private/modules/modulefiles
+  	
+	module use /g/data/u46/users/$USER/devmodules/modulefiles
+
+	module load ard-scene-select-py3-dea/dev_20230523
+	# module load ard-scene-select-py3-dea/20230522
+
+fi
+
+dry_run=" "
+runard="--run-ard"
+runard=""
```

Comment:
hmm. I think we need to revisit this. Can we rid line 21?

Reply:
yes